Sources/ApolloCodegenLib/IR/IR.swift

```diff
@@ -169,7 +169,7 @@ class IR {
 
         var newline: String
         for fragment in referencedFragments {
-          newline = #"\n"#
+          newline = "\n"
```

Comment:
Ok so did a little bit of testing (thanks @calvincestari for calling out the raw string potentially being an issue) and believe the issue is the operation string in the json that is getting processed with the `\n` treated as an actual newline, but because we switched this piece of code in hash generation to a raw string it was being processed as the actual characters. After discussion with @AnthonyMDev this should fix the actual issue while maintaining backward compatibility for legacy apqs